UnityProject/Assets/Scripts/InGameEvents/InGameEventsManager.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using DiscordWebhook;
+
+namespace InGameEvents
+{
+	/// <summary>
+	/// The controller for in game events
+	/// </summary>
+	public class InGameEventsManager : MonoBehaviour
+	{
+		private static InGameEventsManager instance;
+		public static InGameEventsManager Instance => instance;
+
+		private float Timer = 0f;
```

Comment:
Please, check the naming, so that all private variables and non-public fields except for constants / constant-like ones would be in camelCase and properties, public fields, classes, structs, enum constants, and methods would be in PascalCase
https://unitystation.github.io/unitystation/contribution-guides/Development-Standards-Guide/#code-style

Reply:
think ive fixed them all now.